Bottom: fa8fc06a514853010b399cbb2a3970b5660a2375
Top:    3f9f7db6bbdd9ee46d84022abb977eac09f1b7c7
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-27 17:12:34 -0500

Refresh of sshd-create-role

---
diff --git a/roles/sshd/README.md b/roles/sshd/README.md
deleted file mode 100644
index 225dd44b..00000000
--- a/roles/sshd/README.md
+++ /dev/null
@@ -1,38 +0,0 @@
-Role Name
-=========
-
-A brief description of the role goes here.
-
-Requirements
-------------
-
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.
-
-Role Variables
---------------
-
-A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role. Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
-
-Dependencies
-------------
-
-A list of other roles hosted on Galaxy should go here, plus any details in regards to parameters that may need to be set for other roles, or variables that are used from other roles.
-
-Example Playbook
-----------------
-
-Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
-
-    - hosts: servers
-      roles:
-         - { role: username.rolename, x: 42 }
-
-License
--------
-
-BSD
-
-Author Information
-------------------
-
-An optional section for the role authors to include contact information, or a website (HTML is not allowed).
diff --git a/roles/sshd/handlers/main.yml b/roles/sshd/handlers/main.yml
deleted file mode 100644
index 13434437..00000000
--- a/roles/sshd/handlers/main.yml
+++ /dev/null
@@ -1,2 +0,0 @@
----
-# handlers file for sshd
\ No newline at end of file
diff --git a/roles/sshd/meta/main.yml b/roles/sshd/meta/main.yml
deleted file mode 100644
index 72237997..00000000
--- a/roles/sshd/meta/main.yml
+++ /dev/null
@@ -1,57 +0,0 @@
-galaxy_info:
-  author: your name
-  description: your description
-  company: your company (optional)
-
-  # If the issue tracker for your role is not on github, uncomment the
-  # next line and provide a value
-  # issue_tracker_url: http://example.com/issue/tracker
-
-  # Some suggested licenses:
-  # - BSD (default)
-  # - MIT
-  # - GPLv2
-  # - GPLv3
-  # - Apache
-  # - CC-BY
-  license: license (GPLv2, CC-BY, etc)
-
-  min_ansible_version: 1.2
-
-  # If this a Container Enabled role, provide the minimum Ansible Container version.
-  # min_ansible_container_version:
-
-  # Optionally specify the branch Galaxy will use when accessing the GitHub
-  # repo for this role. During role install, if no tags are available,
-  # Galaxy will use this branch. During import Galaxy will access files on
-  # this branch. If Travis integration is configured, only notifications for this
-  # branch will be accepted. Otherwise, in all cases, the repo's default branch
-  # (usually master) will be used.
-  #github_branch:
-
-  #
-  # platforms is a list of platforms, and each platform has a name and a list of versions.
-  #
-  # platforms:
-  # - name: Fedora
-  #   versions:
-  #   - all
-  #   - 25
-  # - name: SomePlatform
-  #   versions:
-  #   - all
-  #   - 1.0
-  #   - 7
-  #   - 99.99
-
-  galaxy_tags: []
-    # List tags for your role here, one per line. A tag is a keyword that describes
-    # and categorizes the role. Users find roles by searching for tags. Be sure to
-    # remove the '[]' above, if you add tags to this list.
-    #
-    # NOTE: A tag is limited to a single word comprised of alphanumeric characters.
-    #       Maximum 20 tags per role.
-
-dependencies: []
-  # List your role dependencies here, one per line. Be sure to remove the '[]' above,
-  # if you add dependencies to this list.
\ No newline at end of file
diff --git a/roles/sshd/tests/inventory b/roles/sshd/tests/inventory
deleted file mode 100644
index 878877b0..00000000
--- a/roles/sshd/tests/inventory
+++ /dev/null
@@ -1,2 +0,0 @@
-localhost
-
diff --git a/roles/sshd/tests/test.yml b/roles/sshd/tests/test.yml
deleted file mode 100644
index 6674bff3..00000000
--- a/roles/sshd/tests/test.yml
+++ /dev/null
@@ -1,5 +0,0 @@
----
-- hosts: localhost
-  remote_user: root
-  roles:
-    - sshd
\ No newline at end of file
diff --git a/roles/sshd/vars/main.yml b/roles/sshd/vars/main.yml
deleted file mode 100644
index 70d6160f..00000000
--- a/roles/sshd/vars/main.yml
+++ /dev/null
@@ -1,2 +0,0 @@
----
-# vars file for sshd
\ No newline at end of file
